charts/node-analyzer/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: node-analyzer
 description: Sysdig Node Analyzer
 # currently matching Sysdig's appVersion 1.14.34
-version: 1.29.7
+version: 1.29.8
```

Comment:
Please bump a minor release, this is a behavioural change that should not be included in a patch release 😄